fhir-demo-2023/src/pages/mylab.js

```diff
@@ -21,6 +21,12 @@ export default function Mylab({ location }) {
         </a>
       </h1>
       <Features list={features} />
+      <section id="mylab">
```

Comment:
The info section for each company is intentionally below the event call-to-action areas.

This is to make it easy and fast for event participants to access the info of demo participants at the events.

The event info will be removed after the events, then the company info is the first thing for the readers.